lib/unit-ruby/white_label_language.rb

```diff
@@ -0,0 +1,21 @@
+module Unit
+  class WhiteLabelLanguage < APIResource
+    path '/white-label/language'
+
+    def save_config(config_json)
```

Comment:
Since we’re using this in a few places, perhaps extract into ResourceOperations::SaveConfig